html/pfappserver/root/static.alt/src/views/Users/_components/UsersCreate.vue

```diff
@@ -361,7 +381,10 @@ export default {
             ...{ quiet: true }
           }
           for (let i = 0; i < this.form.multiple.quantity; i++) {
-            const pid = this.form.multiple.prefix + (i + 1)
+            let pid = this.form.multiple.prefix + (i + 1)
+            if (this.domainName) { // append domainName to pid when available (tenant)
+              pid = `${pid}${this.domainName}`
```

Comment:
Shouldn't this be pid@domain instead of piddomain

Reply:
@satkunas Needs to make the change?

Reply:
Yep

Reply:
Suffix delim changed in https://github.com/inverse-inc/packetfence/pull/5348/commits/664d2a88299aacd113353c9002a22dfe38a1b639